services/app-api/src/parameterStore/getStateAnalystEmailsStore.ts

```diff
@@ -0,0 +1,25 @@
+import { Context } from '../handlers/apollo_gql'
+import { logError } from '../logger'
+import { setErrorAttributesOnActiveSpan } from '../resolvers/attributeHelper'
+import { getParameterStore } from './parameterStore'
+
+export const getStateAnalystEmailsStore = async (
+    stateCode: string,
+    span?: Context['span'],
+    operation?: string
+): Promise<string[]> => {
+    const analystsParameterStore = await getParameterStore(
+        `/configuration/${stateCode}/stateanalysts/email`
+    )
+    if (analystsParameterStore instanceof Error) {
+        logError(
```

Comment:
I think this is fine, but it’s interesting to think about where we want to handle this error. The other option would be for this wrapper to reutrn an `| Error` and have the calling code be responsible for deciding to log the error. In that case we’d simplify this wrapper b/c it wouldn’t need the span or the error reporting logic in it. 

Not a case where there’s a clear “right” answer to me but worth considering the tradeoffs.